app/models/micropost.rb

```diff
@@ -0,0 +1,6 @@
+class Micropost < ApplicationRecord
+  # chỉ chấp nhận content tối đa 140 kí tự và không được để trống
```

Comment:
Remove **ALL** comments in this file. Make a note or something that out of your project to keep your knowledges